chart/kafka-connector/README.md

```diff
@@ -65,46 +65,103 @@ $ helm repo update && \
 
 ## Encryption options
 
-1) TLS off (default)
-2) TLS on
+1. TLS off (default)
+2. TLS on
 
 ## Authentication options
 
-1) TLS with SASL using CA from the default trust store
-3) TLS with SASL using a custom CA
-4) TLS with client certificates
+1. TLS with SASL using CA from the default trust store
+2. TLS with SASL using a custom CA
+3. TLS with client certificates
+
+## Async invocations
+
+The connector can be configured to invoke function asynchronously. This lets you use [OpenFaaS async](https://docs.openfaas.com/reference/async/) features like retries.
+To prevent the connector from consuming all Kafka messages at once and submitting them to the OpenFaaS async queue a limit on the number of inflight async invocations can be configured.
+
+Configure the connector for async invocations:
+
+```yaml
+# Invoke functions asynchronously.
+asyncInvocation: true
+
+async:
+  # Limit the number of inflight async invocations for the connector.
+  # A value of 0 indicates no concurrency limit.
+  maxInflight: 0
+
+# Configure an externally-managed NATS server.
+# NATS is used for async invocations and is required when
+# setting the 'async.maxInflight' parameter to a value other than 0.
+# By default the OpenFaaS embedded nats deployment is used.
+nats:
+  external:
+    enabled: false
```

Comment:
A note on setting the external settings as per the values.yaml from the main chart when using an external nats server?

Reply:
A not for this was added.